src/main/webapp/3Dscript.js

```diff
@@ -1,7 +1,32 @@
-// import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
 import {OrbitControls} from 'https://threejs.org/examples/jsm/controls/OrbitControls.js';
-import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
 
+/**
+ * Class for storing 3D coordinates.
+ * @constructor
+ */
+class Point {
+  /**
+   * The constructor for Point.
+   * @param {(number|object)} x An arbitary x data point.
+   * @param {(number|object)} y An arbitary y data point.
+   * @param {(number|object)} z An arbitary z data point.
+   */
+  constructor(x, y, z) {
+    this.x = x;
+    this.y = y;
+    this.z = z;
+  }
+  /**
+   * A getter.
+   * @param {String} axis A string specifying which axis to return;
+   * @return {(number|object)} An arbitary 3D data point.
+   */
+  get(axis) {
+    if (axis == 'x') return this.x;
+    if (axis == 'y') return this.y;
+    if (axis == 'z') return this.z;
+  }
```

Comment:
Wouldn't it be easier just to access `point.x` directly? (It's possible a typedef or record would be more appropriate than a class; I'm not totally sure about JS best practices. But this pattern of translating string to property looks suspect.)